build_runner/lib/src/generate/build.dart

```diff
@@ -83,6 +83,7 @@ Future<BuildResult> build(List<BuildAction> buildActions,
 ///  typically cause a shutdown).
 Future<ServeHandler> watch(List<BuildAction> buildActions,
         {bool deleteFilesByDefault,
+        bool assumeTty,
```

Comment:
we should plumb this through `build` as well

Reply:
Done